tron/core/serviceinstance.py

```diff
@@ -2,17 +2,13 @@
 from __future__ import unicode_literals
 
 import logging
-import operator
```

Comment:
This too. Can this file be safely removed?

Reply:
Yeah, this is the target for next.